flask_rebar/validation.py

```diff
@@ -8,21 +8,12 @@
     :license: MIT, see LICENSE for details.
 """
 from collections import namedtuple
-
-try:
-    from collections.abc import Mapping
-except ImportError:
-    from collections import Mapping
+from collections.abc import Mapping
```

Comment:
I believe this try/except was only required for compatibility with python 2, which we don't support anymore